docs/user-documentation/glossary.md

```diff
@@ -0,0 +1,79 @@
+ 
+## Ingest
+To ingest an object is to add an entry for it in Islandora. This is done primarily through the Drupal interface. 
+ 
+The word 'ingest' is used in repository language because repository software, rather than having an object and associated components forced upon it, takes in and processes the object itself, creating derivatives automatically. Often, when referring to the process of ingesting objects, Islandora and its associated documentation use other terms such as 'import' or 'add resource node'. In such contexts, these terms generally refer to the ingest process.
+ 
+## Media
+A drop-in replacement for the Drupal core upload field with a unified User Interface where editors and administrators can upload, manage, and reuse files and multimedia assets. 
+ 
+## Microservice
+A software development technique —a variant of the service-oriented architecture (SOA) structural style— that arranges an application as a collection of loosely coupled services[1] In a microservices architecture, services are fine-grained and the protocols are lightweight.
+ 
+## Open Source
+Open source describes a method of software development that promotes access to the end product's source code. Islandora is an open source product with an active development community, operating under the GPL license (2.0) for Drupal components and the MIT license for non-Drupal components. 
+ 
+## Module
+Software (usually PHP, JavaScript, and/or CSS) that extends site features and adds functionality. The Drupal project distinguishes between core and contributed modules.
+ 
+## Node
+A node is any piece of individual content, such as a page, poll, article, forum topic, or a blog entry. All content on a Drupal website is stored and treated as "nodes". For information about nodes specific to Islandora, see "Resource Node."
+ 
+## Resource Node
+A resource node is a Drupal node that represents a single conceptual item or object stored in an Islandora repository. It acts as a container for all files and metadata associated with that item, and is the place where the item 'lives' as a visitable url. 
+ 
+For example, A video stored in Islandora will have a resource node, which metadata stored in fields, a preservation master file, and may have a thumbnail, web-friendly derivatives, and technical metadata associated with the resource node. The resource node may also belong to one or more collections. 
+ 
+## Role
+A named set of permissions that can be applied to a user account. 
+ 
+## Taxonomy term
+A term used to classify content, such as a tag or a category. 
+ 
+## Theme
+Software and asset files (images, CSS, PHP code, and/or templates) that determine the style and layout of the site. The Drupal project distinguishes between core and contributed themes. 
+ 
+## View 
+A Drupal term for a formatted listing of data; typically, the data comes from content entities.
```

Comment:
"An abstract query configuration used to generate lists and tables of content entities or specific content entity fields. Drupal provides a powerful administrator interface for creating and editing View configurations."